pkg/collector/corechecks/ebpf/oom_kill.go

```diff
@@ -109,18 +110,24 @@ func (m *OOMKillCheck) Run() error {
 		return log.Errorf("Raw data has incorrect type")
 	}
 	for _, line := range oomkillStats {
-		entityID := containers.BuildTaggerEntityName(line.ContainerID)
+		containerID, err := cgroups.ContainerFilter("", line.CgroupName)
```

Comment:
Should we remove the useless first parameter of [this function](https://github.com/DataDog/datadog-agent/blob/6fd8cb9e0dccb23be36e407d05065f684ac05d04/pkg/util/cgroups/reader.go#L51)?

Reply:
We can't it implements the generic `ReaderFilter` type